src/plugins/external_tool_support/src/blast/BlastCommonTask.cpp

```diff
@@ -238,6 +249,9 @@ void BlastCommonTask::parseTabularLine(const QByteArray &line) {
         stateInfo.setError(tr("Incorrect number of fields in line: %1").arg(elements.size()));
         return;
     }
+    int querySequenceIndex = parseQuerySequenceIndex(elements[0]);
```

Comment:
Could be crash is [0] isn't exists, add `SAFE_POINT` or use `->value(0);` if the default value is possible

Reply:
Could you check 3 lines above this line: if (elements.size() != 12) {....

I would discourage anyone to use value() in vectors/list because this is the way to hide real bugs: crash is better than incorrectly computed result.